SoftLayer/CLI/globalip/assign.py

```diff
@@ -5,17 +5,17 @@
 
 import SoftLayer
 from SoftLayer.CLI import environment
-from SoftLayer.CLI import helpers
 
 
-@click.command()
+@click.command(epilog="More information about types and ")
 @click.argument('identifier')
-@click.argument('target')
+@click.option('--target',
+              help='See SLDN docs. '
+                   'E.g SoftLayer_Network_Subnet_IpAddress, SoftLayer_Hardware_Server,SoftLayer_Virtual_Guest')
```

Comment:
Lets change this to a [click.choice](https://click.palletsprojects.com/en/8.0.x/options/#choice-options) option, and list out all the types in a short-hand format

so something like `type=click.Choice['vlan', 'ip', 'hardware', 'vsi' ]` Then the  network manager function can figure out which SoftLayer class needs to be sent in.

Both this and `--target-id` should be marked as required as well.